test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -1413,6 +1413,8 @@ var Annotations = map[string]string{
 
 	"[sig-node-tuning] NTO should OCP-66086 NTO Prevent from stalld continually restarting [Slow]": "",
 
+	"[sig-node-tuning] NTO should SNO installation does not finish due to wait for non-existing machine-config [Early]": " [Suite:openshift/conformance/parallel]",
```

Comment:
I hope the test case execute once the cluster is ready, just in case it will not impact by other test case, the [Early] means execute early than other test cases.  because our goal is verify the sno installation failed by NTO and mcp